overlord/hookstate/hookmgr_test.go

```diff
@@ -167,6 +232,34 @@ func (s *hookManagerSuite) TestHookTaskHandlerDoneError(c *C) {
 	checkTaskLogContains(c, s.task, regexp.MustCompile(".*done failed at user request.*"))
 }
 
+func (s *hookManagerSuite) TestHookTaskHandlerErrorError(c *C) {
```

Comment:
Heh, it's handling an error from the handler's `Error()` function, i.e. this test is checking that an `Error()` error is handled.